retrofit-converters/gson/src/test/java/retrofit2/converter/gson/GsonConverterFactoryTest.java

```diff
@@ -78,7 +78,7 @@
   }
 
   interface Service {
-    @POST("/") Call<AnImplementation> anImplementation(@Body AnImplementation impl);
+    @POST("/") Call<AnImplementation> anImplementation(@Body(ignoreNull = false) AnImplementation impl);
```

Comment:
The boolean controls whether or not `null` is passed to the converter or whether it just becomes an empty body and skips the converter.

Maybe `convertNull` then?